mlprimitives/jsons/sklearn.linear_model.Lasso.json

```diff
@@ -102,27 +94,33 @@
                 "description" : "Whether to calculate the intercept for this model. If set to False, no intercept will be used in calculations (e.g. data is expected to be already centered)."
             },
             "alpha": {
-                "type": "multitype",
-                "description": "Regularization strength. Alpha corresponds to C^-1 in other linear models such as LogisticRegression or LinearSVC.",
-                "types": {
-                    "float":{
-                        "description": "Must be a positive float. Larger values specify stronger regularization"
-                    },
-                    "array":{
-                        "description": "If an array is passed, penalties are assumed to be specific to the targets. Hence they must correspond in number."
-                    }
-                },
-                "default": 1.0
+                "type": "float",
+                "description": "Regularization strength. Alpha corresponds to C^-1 in other linear models such as LogisticRegression or LinearSVC. Must be a positive float.",
+                "default": 1.0,
+                "range": [0.0, 10.0]
```

Comment:
Oh yes I thought if alpha was  0 then it would just be normal regression but in general it's really not recommended so I changed it. Thanks so much for pointing this out.